final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,57 @@
+from urllib.error import URLError
+from final_task.rss_reader.scripts import pars_args
+from final_task.rss_reader.scripts import parser_rss
+import logging
+import sys
+
+
+def main():
+    """
+        The main entry point of the application
+    """
+    try:
+        args = pars_args.get_args()
+        logger = logging.getLogger("rss_reader")
+        logger.setLevel(logging.INFO)
+        # create the logging file handler
+        if not args.verbose:
+            fh = logging.FileHandler("new_snake.log")
+            formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+            fh.setFormatter(formatter)
+            logger.addHandler(fh)
+        else:
+            fh = logging.basicConfig(stream=sys.stdout,
+                                     filemode='a',
+                                     format='%(asctime)s,%(msecs)d %(name)s %(levelname)s %(message)s',
+                                     datefmt='%H:%M:%S',
+                                     level=logging.DEBUG)
+
+        # add handler to logger object
+
+        logger.info("Program started")
+
+        list_of_news = []
+        news_feed = parser_rss.get_news_feed(args.source)
+        parser_rss.init_list_of_news(list_of_news, news_feed, args.limit)
+        if args.json:
+            parser_rss.print_news_in_json(list_of_news)
+        else:
+            parser_rss.print_news(list_of_news)
+
+    except ValueError:
+        raise
+    except URLError:
+        raise
+    except parser_rss.TimeOutExeption:
+        raise
```

Comment:
You don't really need to do try-except in this function if all you are going to do is to reraise the exception.